specification/maps/data-plane/Timezone/preview/1.0/timezone.json

```diff
@@ -538,7 +551,7 @@
           "readOnly": true
         },
         "hasZone1970Location": {
-          "description": "HasZone1970Location property",
+          "description": "This represents the property which the zone has clocks agreeing since 1970",
```

Comment:
The wording here is confusing to me.

Reply:
Hi @tjprescott , I have updated the description. Hopefully this will make better sense to you and users. 
Thanks again for the feedback.